stable/gangway/templates/deployment.yaml

```
@@ -35,6 +35,9 @@ spec:
         - -config
         - /gangway/gangway.yaml
         env:
+{{ if .Values.env -}}
```

Changing this to:
```suggestion
{{- if .Values.env }}
```

Means the `yaml` doesn't have a blank line when `env` isn't set. Trivial I'll admit but slightly nicer reading for anybody previewing the `yaml`!